### ISSUE TITLE
unittest2 0.5.1 fails tests with Python 3.3

### ISSUE THREAD

**Issue by GoogleCodeExporter** [testing-cabal/unittest-ext#65]
```
unittest2 0.5.1 fails 2 tests with Python 3.3 due to changed exception messages.

................................................................................
.................................F..................F...........................
................................................................................
......................................
======================================================================
FAIL: test_loadTestsFromName__unknown_module_name 
(unittest2.test.test_loader.Test_TestLoader)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/var/tmp/portage/dev-python/unittest2-0.5.1/work/unittest2-0.5.1-3.3/unittest2/test/test_loader.py", line 240, in test_loadTestsFromName__unknown_module_name
    loader.loadTestsFromName('sdasfasfasdf')
ImportError: No module named 'sdasfasfasdf'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/var/tmp/portage/dev-python/unittest2-0.5.1/work/unittest2-0.5.1-3.3/unittest2/test/test_loader.py", line 242, in test_loadTestsFromName__unknown_module_name
    self.assertEqual(str(e), "No module named sdasfasfasdf")
AssertionError: "No module named 'sdasfasfasdf'" != 'No module named 
sdasfasfasdf'
- No module named 'sdasfasfasdf'
?                 -            -
+ No module named sdasfasfasdf


======================================================================
FAIL: test_loadTestsFromNames__unknown_module_name 
(unittest2.test.test_loader.Test_TestLoader)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/var/tmp/portage/dev-python/unittest2-0.5.1/work/unittest2-0.5.1-3.3/unittest2/test/test_loader.py", line 620, in test_loadTestsFromNames__unknown_module_name
    loader.loadTestsFromNames(['sdasfasfasdf'])
ImportError: No module named 'sdasfasfasdf'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/var/tmp/portage/dev-python/unittest2-0.5.1/work/unittest2-0.5.1-3.3/unittest2/test/test_loader.py", line 622, in test_loadTestsFromNames__unknown_module_name
    self.assertEqual(str(e), "No module named sdasfasfasdf")
AssertionError: "No module named 'sdasfasfasdf'" != 'No module named 
sdasfasfasdf'
- No module named 'sdasfasfasdf'
?                 -            -
+ No module named sdasfasfasdf


----------------------------------------------------------------------
Ran 278 tests in 0.343s

FAILED (failures=2)
```

Original issue reported on code.google.com by `Arfrever...@gmail.com` on 22 Jun 2011 at 11:58

**Comment by GoogleCodeExporter** [comment]
```
this appears to relate to Python issue # 8754 [1], which requested 
quote-enclosure of missing module names in the exception message.  

[1]: http://bugs.python.org/issue8754
```

Original comment by `ReubenGa...@gmail.com` on 9 Dec 2013 at 3:21

**Comment by GoogleCodeExporter** [comment]
```
Here's a patch that fixes this:

https://code.google.com/r/msabramo-unittest-ext/source/detail?r=56585a6ec1f53fe2
885e5372c23849ee9b3e316e&name=py33_and_py34
```

Original comment by `msabr...@gmail.com` on 5 May 2014 at 3:17

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 0.7.0
```

Original comment by `robert.c...@gmail.com` on 4 Nov 2014 at 3:17
- Changed state: **Fixed**
